### PR TITLE
order list items for output

### PR DESCRIPTION
**Pull request by thutterer**
- products
- repos
- custom reps

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Unless this is a trivial change, please include a CHANGELOG entry.<br />
Run <code>osc vc</code> in the <code>package</code> directory to add one.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: lib/rmt/cli/repos.rb [hunk 1]
```diff
@@ -28,7 +28,7 @@ def disable(id)
   protected
 
   def list_repositories(scope: :enabled)
-    repositories = (scope == :all) ? Repository.only_scc : Repository.only_scc.only_mirrored
+    repositories = ((scope == :all) ? Repository.only_scc : Repository.only_scc.only_mirrored).order(:name, :description)
```
Comment (ikapelyukhin) [inline lib/rmt/cli/repos.rb]: Why `:description`? Maybe there's something more suitable for sorting, arch or friendly name maybe?

Comment (thutterer) [reply lib/rmt/cli/repos.rb]: Well, it only makes sense to order by stuff that is also visible in the output, which would be:
https://github.com/SUSE/rmt/blob/3f45944df9ed56e37d052fda544e967860b563a7/lib/rmt/cli/repos.rb#L40-L47

Comment (ikapelyukhin) [reply lib/rmt/cli/repos.rb]: Well, description is basically `#{name} for #{arch}`, so sorting by description is equivalent to sorting by arch when names are the same.

**Comment by thutterer** [comment]
> Please mention the bug number on the changes file.

@ikapelyukhin Done. I amended the previous commit with the changelog entry.

**Pull request merged**